src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
```

Comment:
The name of the array should be in camelCase, starting with a lowercase letter, for example, 'arrayOfFigures'.